.github/workflows/build-container.yml

```diff
@@ -0,0 +1,81 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [make-date-tag]
+    strategy:
+      fail-fast: False
+      matrix:
+        dver: ['7', '8']
+        repo: ['devel', 'testing', 'release']
+        series: ['3.5', '3.6']
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate tag list
+      id: generate-tag-list
+      env:
+        EL_VER: ${{ matrix.dver }}
+        BASE_YUM_REPO: ${{ matrix.repo }}
+        OSG_RELEASE: ${{ matrix.series }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
+      run: |
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tags=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO-el$EL_VER )
+          tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
```

Comment:
Another race condition, we only want to assign this tag if `BASE_YUM_REPO == release`

Reply:
Are you referring to the tag in line 45?

Reply:
Yup! For example, `3.6-el8` should correspond to `3.6-release-el8`, not `3.6-testing-el8` or `3.6-devel-el8`